packages/loot-core/src/platform/server/fetch/index.electron.ts

```diff
@@ -1,2 +1,12 @@
-// @ts-strict-ignore
-export { default as fetch } from 'node-fetch';
+// // @ts-strict-ignore
+import nodeFetch from 'node-fetch';
+
+export const fetch = (input: RequestInfo | URL, options?: RequestInit) => {
+  return nodeFetch(input, {
+    ...options,
+    headers: {
+      ...options?.headers,
+      origin: 'app://actual',
```

Comment:
The issue was that the forked server process in electron isn't able to pass along the browser window information (like the origin etc). 

The gocardless endpoint needs the origin.

If anyone can think of a better way to do this I'm all ears. 

I think this has been broken for quite some time. The old version of the app had a server.html which it was loading in a separate browser window - it might have worked then because the server process was running in browser context. We moved away from that a long time ago.